docs/source/deployment/deploy-model-to-kubernetes/tutorial.rst

```diff
@@ -0,0 +1,484 @@
+
+There are two ways to specify the model for deployment in KServe configuration file:
+
+1. Build a Docker image with the model and specify the image URI.
+2. Specify the model URI directly (this only works if your model is stored in remote storage).
+
+Please open the tabs below for details on each approach.
+
+
+.. tabs::
+
+  .. tab:: Using Docker Image
+
+    .. raw:: html
+
+      <h4>Register Docker Account</h4>
+
+    Since KServe cannot resolve locally built Docker image, you need to push the image to a Docker registry.
+    For this tutorial, we'll push the image to `Docker Hub <https://hub.docker.com/>`_, but you can use any other Docker registry,
+    such as `Amazon ECR <https://aws.amazon.com/ecr/>`_ or private registry.
+
+    If you don't have a Docker Hub account yet, create one at https://hub.docker.com/signup.
+
+    .. raw:: html
+
+      <h4>Build a Docker Image</h4>
+
+    Build a ready-to-deploy Docker image with the ``mlflow models build-docker`` command:
+
+    .. code-block:: bash
+
+      mlflow models build-docker -m runs:/<run_id_for_your_best_run>/model -n <your_dockerhub_user_name>/mlflow-wine-classifier --enable-mlserver
+
+    This command builds a Docker image with the model and dependencies, tagging it as ``mlflow-wine-classifier:latest``.
+
+    .. raw:: html
+
+      <h4>Push the Docker Image</h4>
+
+    After building the image, push it to Docker Hub (or to another registry using the appropriate command):
+
+    .. code-block:: bash
+
+      docker push <your_dockerhub_user_name>/mlflow-wine-classifier
+
+    .. raw:: html
+
+      <h4>Write Deployment Configuration</h4>
+
+    Then create a YAML file like this:
+
+    .. code-block:: yaml
+
+      apiVersion: "serving.kserve.io/v1beta1"
+      kind: "InferenceService"
+      metadata:
+        name: "mlflow-wine-classifier"
+        namespace: "mlflow-kserve-test"
+      spec:
+        predictor:
+          containers:
+            - name: "mlflow-wine-classifier"
+              image: "<your_docker_user_name>/mlflow-wine-classifier"
+              ports:
+                - containerPort: 8080
+                  protocol: TCP
+              env:
+                - name: PROTOCOL
+                  value: "v2"
+
+
+  .. tab:: Using Model URI
+
+    .. raw:: html
+
+      <h4>Get Remote Model URI</h4>
+
+    KServe configuration allows direct specification of the model URI. However, it doesn't resolve MLflow-specific URI schema like ``runs:/`` and ``model:/``,
+    nor local file URIs like ``file:///``. We need to specify the model URI in a remote storage URI format e.g. ``s3://xxx`` or ``gs://xxx``.
+    By default, MLflow stores the model in the local file system, so you need to configure MLflow to store the model in remote storage.
+    Please refer to `Artifact Store <../../../tracking.html#artifact-stores>`_ for setup instructions.
+
+    After configuring the artifact store, repeat the model training steps.
```

Comment:
or load the best model from the model uri and re-log it from its in-memory object?